src/main/isFeeAccount.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+import * as validator from 'validator'
+
+@ValidatorConstraint()
+export class IsFeeAccountConstraint implements ValidatorConstraintInterface {
+
+  validate (value: any, args: ValidationArguments): boolean {
+    if (value == null) {
+      return true
+    }
+
+    if (typeof value !== 'string') {
+      return false
+    }
+
+    return validator.matches(value.trim(), new RegExp(/^PBA[0-9]{7}$/gi))
+  }
+
+}
+
+/**
+ * Verify a valid fee account number based on the rules:
+ * A Fee Account reference must be in the format PBAnnnnnnn (where n is a number from 0 to 9)
+ * A Fee Account references must have a prefix PBA
```

Comment:
I would say that that comment line and one below are not necessary since we mentioned the format line above.